### PR TITLE
Transition to API v3

### PR DESCRIPTION
**Pull request by krawieck**

### CODE REVIEW

File: lib/widgets/write_comment.dart [hunk 1]
```diff
@@ -83,15 +84,17 @@ class WriteComment extends HookWidget {
       ),
       body: ListView(
         children: [
-          ConstrainedBox(
-            constraints: BoxConstraints(
-                maxHeight: MediaQuery.of(context).size.height * .35),
-            child: SingleChildScrollView(
-              padding: const EdgeInsets.all(8),
-              child: preview,
+          if (preview != null) ...[
```
Comment (shilangyu) [inline lib/widgets/write_comment.dart:87]: analyzing `preview` initialization tells me it is either a `Column` widget or `body` where body is either `SizedBox` or `MarkdownText`, so `preview` is never null. Am I missing something?

**Review by shilangyu — CHANGES_REQUESTED** [review]

File: lib/pages/settings.dart [hunk 2]
```diff
@@ -127,7 +128,7 @@ class AccountsConfigPage extends HookWidget {
             ),
           ) ??
           false) {
-        accountsStore.removeInstance(instanceHost);
+        unawaited(accountsStore.removeInstance(instanceHost));
```
Comment (shilangyu) [inline lib/pages/settings.dart:131]: Why is it unawaited?

**Review by shilangyu — CHANGES_REQUESTED** [review]
`_assignDefaultAccounts` should also return a future

File: lib/pages/settings.dart (hunk 2 above)
Comment (krawieck) [reply lib/pages/settings.dart:131]: cuz it was before, just on the store level

Comment (shilangyu) [reply lib/pages/settings.dart:131]: Before the actions were synchronous and ignored the save, now that it doesn't ignore it it should be correctly awaited

File: lib/pages/manage_account.dart [hunk 3]
```diff
@@ -182,12 +185,13 @@ class _ManageAccount extends HookWidget {
...
 
         try {
-          await LemmyApiV2(user.instanceHost).run(DeleteAccount(
+          await LemmyApiV3(user.instanceHost).run(DeleteAccount(
             password: deleteAccountPasswordController.text,
             auth: token.raw,
           ));
 
-          accountsStore.removeAccount(user.instanceHost, user.name);
+          unawaited(
```
Comment (shilangyu) [inline lib/pages/manage_account.dart:193]: same here, no reason for unawaited

**Review by shilangyu — CHANGES_REQUESTED** [review]

File: lib/pages/settings.dart (hunk 2 above)
Comment (shilangyu) [reply lib/pages/settings.dart:131]: I thought that was the point of this change, to make sure we await for those

**Pull request merged**